client/src/view/gotchi-view.tsx

```diff
@@ -116,7 +116,7 @@ export class GotchiView extends React.Component<IGotchiViewProps, IGotchiViewSta
                                 />
                             </R3.Object3D>
                         )}
-                        {!journey ? null : (
+                        {!journey ? undefined : (
```

Comment:
ah yes.  cool to be more consistent.

Reply:
found a few more of these, fixed 'em in your branch

Reply:
:) thx